src/nxdoc/nuxeo-server/administration/configuration-parameters-index-nuxeoconf.md

```diff
@@ -826,7 +826,7 @@ Since Nuxeo 5.8\. Database timeout after which connections not in use are remove
 
 **Since Nuxeo 5.8**
 
-Since Nuxeo 5.8\. Whether full text indexing and querying should be completely disabled in the repository. See [VCS]({{page version='' space='nxdoc' page='vcs'}}) for details.
+Since Nuxeo 5.8\. Whether full text indexing and querying should be completely disabled in the repository. It also disable the **FulltextExtractorWork**. See [VCS]({{page version='' space='nxdoc' page='vcs'}}) for details. Can also be applied to [DBS]({{page version='' space='nxdoc' page='dbs'}}).
```

Comment:
disable -> disables
You shouldn't talk about the work instance that's disabled, but just mention that it "disables fulltext extraction"

Reply:
Ok, maybe that's not the good format to talk about this topic. I have been very frustrated that when looking on the internet to disable the FulltextExtractorWork that nothing comes up. The only solution is dig the code base and try to  figure it out and spend a couple of hours.